src/style/app.scss

```diff
@@ -586,3 +579,8 @@ h3 {
   overflow-wrap: break-word;
   word-wrap: break-word;
 }
+
+.mdi {
```

Comment:
@tamazlykar Do we need this style? You didn't add this class to any html tag. MDI library itself has default font size 24px

Reply:
@vanekbr Yes, we need it. Because icons implements by font, they rely on font-size attribute. Angular Material have rule .mat-icon-button that define font-size at 14 px, so because of this we need to provide our own desired size for icons.
This .mdi class applies to icons in indirect way. We can register class that will be applied to every <mat-icon> tag in application. It registered in app.module (link below), also MDI library require .mdi class for every icon to properly render icon.

https://github.com/tamazlykar/cloudstack-ui/blob/aba1ab988728d0792caa6b0ec219d81cfd343b22/src/app/app.module.ts#L143-L145